compiler/luci/pass/src/FuseAddWithTConvPass.cpp

```diff
@@ -105,6 +107,9 @@ bool fuse_add_with_tconv(luci::CircleTransposeConv *tconv)
     replace(add).with(tconv);
   }
 
+  // set origin
+  luci::add_origin(tconv, luci::get_origin(add));
```

Comment:
Q) I thought new nodes are set with origin. `tconv` is existing node. is there a rule for setting origin where I can understand the logic?

Reply:
@seanshpark Basically, origin propagation is for preserving original nodes' information. If node A and node B are fused into new node C, node C should be inherited node A's and node B's origin. Likewise, if node A is fused to node B(no new node, just node B absorbed node A), then node B should get node A's origin. "preserving original nodes" would be helpful.

Reply:
Thanks for the explanation. It seems reasonable.
And is there any document for readers or guide for writers to follow?

Reply:
> Well, no. Actually, it's a bit hard to find proper place to document.``)

Ah.. how about in `luci/Profile/CircleNodeOrigin.h` file? as this is another task, you can do this later or post an task issue not to forget :)